systemd/openqa-dehydrated.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Renewal openQA certificates.
+
+[Timer]
+OnCalendar=*-*-* 00:00:00
```

Comment:
I think it's a bad time to start exactly then. Better select a random time to not have multiple processes interfering

Reply:
sure but I just copy what the crontab shows

Reply:
If you want a 2h random time window, you can use for example this one:
```
OnCalendar=daily
RandomizedDelaySec=7200
```